slackbridge/bots.py

```diff
@@ -149,6 +151,30 @@ def nickserv_auth(self):
                 self.target_group_nick,
                 self.nickserv_password,
             ))
+    def privmsg(self, user, channel, message):
+        """
+        Set to handle if channel is own name (private chat)
+        """
+
+        nick = utils.nick_from_irc_user(user)
+        
+        if channel == self.nickname:
+            if not self.im_id:
+                self.im_id = self.sc.api_call(
+                        'im.open',
+                        user=self.user_id,
+                        return_im=True
+                        )['channel']['id']
```

Comment:
I'd unindent this line to make it clearer this is actually a function call (and add a trailing comma):

```
self.im_id = self.sc.api_call(
    'im.open',
    user=self.user_id,
    return_im=True,
)['channel']['id']
```

Or even just splitting into one variable for the return value of the API call and another to actually index, just for clarity.